npc/re/merchants/ammo_boxes.txt

```diff
@@ -167,8 +167,8 @@ S_Sphere_Exchange:
 				case 1:
 					mes "[Kenny]";
 					mes "All done.";
-					delitem .@slug_id, .@new_count * 10;
-					getitem .@new_slug_id, .@old_count;
+					delitem .@old_id, .@new_count * 10;
```

Comment:
Should also be done above, where it is output via "mes" or not?
There `.@old_count` is used directly.